### PR TITLE
Optimization by P

### PR DESCRIPTION
**Pull request by aangerma**

### CODE REVIEW

File: src/algo/depth-to-rgb-calibration/calibration-types.h [hunk 1]
```diff
@@ -27,6 +28,66 @@ namespace depth_to_rgb_calibration {
         bool operator != (const double2 d) { return !(*this == d); }
     };
 
+	struct double3x3
+	{
+		double mat[3][3];
```
Comment (maloel) [inline src/algo/depth-to-rgb-calibration/calibration-types.h:33]: Tabs...

File: src/algo/depth-to-rgb-calibration/calibration-types.h [hunk 2]
```diff
@@ -27,6 +28,66 @@ namespace depth_to_rgb_calibration {
...
+		double3x3 transpose()
+		{
+			double3x3 res = { 0 };
+
+			for (auto i = 0; i < 3; i++)
```
Comment (maloel) [inline src/algo/depth-to-rgb-calibration/calibration-types.h:38]: If it's a simple 3x3, three swaps can do it...

File: unit-tests/algo/depth-to-rgb-calibration/compare-scene.h [hunk 3]
```diff
@@ -127,10 +128,15 @@ void compare_scene( std::string const & scene_dir )
     auto cb = [&]( algo::iteration_data_collect const & data )
     {
         // data.iteration is 0-based!
-        REQUIRE( data.iteration < md.n_iterations );
+       /* REQUIRE( data.iteration < md.n_iterations );*/
```
Comment (maloel) [inline unit-tests/algo/depth-to-rgb-calibration/compare-scene.h:131]: Why?

**Review by maloel — APPROVED** [review]

File: unit-tests/algo/depth-to-rgb-calibration/compare-scene.h (hunk 3 above)
Comment (aangerma) [reply unit-tests/algo/depth-to-rgb-calibration/compare-scene.h:131]: removed by mistake I will return it .

**Pull request merged**